docs/build_version_doc/setup_docs_ubuntu.sh

```diff
@@ -0,0 +1,42 @@
+# If you need to build <= v0.12.0 then use a Python 2 environment
+# mxdoc.py - a sphinx extension, was not Python 3 compatible in the old versions
+# source activate mxnet_p27
+
+# Install dependencies
+sudo apt-get update
```

Comment:
Please move this code into a Dockerfile instead of a shell script.  See https://github.com/apache/incubator-mxnet/blob/master/docs/Dockerfile for reference

Reply:
This is taking a really long time to test, so that's why an update hasn't landed yet! I had the benefit of some deps handled by the DL AMI, but in a vanilla container I had a lot of troubleshooting to do. That being said, I'm hoping I'm near the end of testing the scripts inside a container for every tag we need.

I'm quite tempted to modify the scripts though so that they take in the tag as an input and we do a `RUN` for each, so we can use the docker cache for when the build steps are completed fine.

Reply:
It would be good if the tag(s) can be passed as argument. E.g. "docker run Dockerfile_doc -e tags=1.1.0,1.0.0"

Reply:
Btw, please remove this file as it is no longer needed

Reply:
I'd like to keep it around... or maybe move the info to documentation since it is really helpful for installing outside of docker.

Reply:
Feel free to move the instructions into a README.md.

Reply:
Is this file deprecated? Please remove